src/snowflake/snowpark/modin/plugin/_internal/resample_utils.py

```diff
@@ -51,6 +51,8 @@
     "var",
     "count",
     "size",
+    "first",
+    "last",
```

Comment:
Adding to list of implemented Resampler APIs ensures the tests for these API run.